app/src/main/java/fr/free/nrw/commons/upload/FileProcessor.java

```diff
@@ -104,8 +107,10 @@ GPSExtractor processFileCoordinates(SimilarImageInterface similarImageInterface,
      * @return        tags to be redacted
      */
     private Set<String> getExifTagsToRedact(Context context) {
-        Type setType = new TypeToken<Set<String>>() {}.getType();
-        Set<String> prefManageEXIFTags = defaultKvStore.getJson(Prefs.MANAGED_EXIF_TAGS, setType);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+        Set<String> prefManageEXIFTags = sharedPreferences.getStringSet("manageExifTags", new HashSet<String>());
+        //Type setType = new TypeToken<Set<String>>() {}.getType();
```

Comment:
remove commented code if its not being used.

Reply:
Sure